src/chart-configuration/range-timeseries-data-configuration.ts

```diff
@@ -0,0 +1,162 @@
+export const RANGEDATA_GROUPS_CONFIGURATION = [
+  {
+    type: "hosp_need",
```

Comment:
I've given similar comment before. Don't use a contract that is very front-end/highchart related (e.g. xAxisLabel). It should be set by the front-end instead of the backend. Imagine if we need to show another label, we need to change backend which doesn't much sense.

Reply:
decoupling the backend with the charting library can make it easier for the front-end to make adjustments. Last Friday I tried to change the title of the line chart and found that changing it is not very straightforward.

Reply:
hmm, yah i pinged on slack about this, lets discuss this a bit further,